tests/spec/core/utils-spec.js

```diff
@@ -375,6 +379,22 @@ describe("Core - Utils", () => {
         })
       ).toEqual("X, X, X, and X");
     });
+    it("joins with proper commas and 'and' when Intl.ListFormat is available", () => {
```

Comment:
Same as above, all that matters is that the right output is produced, not that the intl API is involved.